DEVGUIDE.md

```diff
@@ -152,13 +152,13 @@ components installed into Visual Studio 15.  You can revert this step by disabli
 
 For **Debug**:
 
-1. Ensure that the VSIX package is uninstalled. In VS, select Tools/Extensions and Updates and if the package `VisualStudio.FSharp.EnableOpenSource` is installed, select Uninstall
-1. Run ``debug\net40\bin\EnableOpenSource.vsix``
+1. Ensure that the VSIX package is uninstalled. In VS, select Tools/Extensions and Updates and if the package `Visual F# Tools` is installed, select Uninstall
+1. Run ``debug\net40\bin\VisualFSharpVsix.vsix``
```

Comment:
Thanks for the other comments below.  Could you rewrite sections 4 and 5 of DEVGUIDE.md to tell the developer what they need to do to use the freshly compiled bits (either add it to this PR or a direct edit to DEVGUIDE.md)?  e.g. what is the MSI for, can they uninstall it etc.

thanks!